### ISSUE TITLE
Error when providing options - "values is not defined"

### ISSUE THREAD

**Issue by reaganchisholm** [designsystemsinternational/mechanic#116]
**Describe the bug**

```
ReferenceError: values is not defined - You may find more context in console.
```

**To Reproduce**
1. Add options to input to convert it to select field type

```js
export const inputs = {
  color: {
    type: "number",
    default: 1,
    options: [1, 5, 10], 
  },
  // ...
}
```

**Expected behavior**
Select dropdown should be provided with options without rendering error.

**Screenshots**
![image](https://user-images.githubusercontent.com/19511364/158927502-432f40c8-f9eb-4352-a969-53dd2919d2a6.png)

**Desktop**
 - Windows
 - Chrome
 - 2.0.0-beta.1

**Additional context**
I tested this out on version `1.2.0` and it works as expected so it seems related to most recent `2.0.0-beta` release.

**Comment by fdoflorenzano** [comment]
Thanks for this! We'll try to fix it as soon as possible.

**Comment by fdoflorenzano** [comment]
Hey @reaganchisholm! We just released `v2.0.0-beta.2`, and I think you'll find that the mentioned bug doesn't occur anymore.

**Comment by reaganchisholm** [comment]
Working as expected now, thanks!
